modules/dap/session.py

```diff
@@ -547,6 +559,12 @@ async def step_in(self, granularity: str|None = None):
 	async def step_out(self, granularity: str|None = None):
 		await self.step('stepOut', granularity)
 
+	async def step_back(self, granularity: str|None = None):
+		await self.step('stepBack', granularity)
+
+	async def record(self):
```

Comment:
I think this should live in the gdb adapter its not part of the dap spec

Reply:
done